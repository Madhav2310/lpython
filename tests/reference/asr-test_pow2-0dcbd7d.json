{
    "basename": "asr-test_pow2-0dcbd7d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow2.py",
    "infile_hash": "9e27335c16e76a61652a253054e35a764d411e0f76bda55d3a6bced2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_pow2-0dcbd7d.stdout",
    "stdout_hash": "f6779aca527ac909f38c941bcf5ccca07db7d726ea8fa6d2bef3d768",
    "stderr": "asr-test_pow2-0dcbd7d.stderr",
    "stderr_hash": "2dc79d784072d4e5606ccf06127df8918e0a23f055dbf0409dd67793",
    "returncode": 0
}