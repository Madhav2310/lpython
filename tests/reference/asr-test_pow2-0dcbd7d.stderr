style suggestion: Could have used '**' instead of 'pow'
 --> tests/errors/test_pow2.py:7:11
  |
7 |     print(pow(a, b))
  |           ^^^^^^^^^ '**' could be used instead
